lms/static/sass/views/_login-register.scss

```diff
@@ -503,6 +507,12 @@ $sm-btn-linkedin: #0077b5;
         }
     }
 
+    .institution-list {
+        li {
```

Comment:
Please do not use HTML elements as styling selectors. Switch this to a class-base selector.